### ISSUE TITLE
On licensing

### ISSUE THREAD

**Issue by cdancy** [MaksymBilenko/docker-oracle-12c#97]
Are there any licensing issues to be had when using this say for desktop or devops related purposes?

**Comment by buamod** [comment]
Same question ? what is the license used here ?

**Comment by theothermattm** [comment]
Seems as though this definitely could have licensing issues: https://github.com/MaksymBilenko/docker-oracle-12c/issues/128

**Comment by cdancy** [comment]
Interesting... _technically_ there is on issue IF you have the proper licensing but oracle is so wishy-washy as to how they define things that causes more confusion than anything else.

**Comment by theothermattm** [comment]
Agreed, but I'm guessing it's the same reason why they make you click a radio button to agree to their terms before you download anything on their site 👎
